client/my-sites/hosting/sftp-card/index.js

```diff
@@ -159,8 +161,8 @@ const SftpCard = ( {
 				<div className="sftp-card__questions">
 					<Accordion title={ translate( 'What is SFTP?' ) }>
 						{ translate(
-							'It’s a secure way for you to access your website files on your local computer via a client program such as {{a}}Filezilla{{/a}} ' +
-								'For more information see {{supportLink}}SFTP on WordPress.com{{/supportLink}} ',
+							'It’s a secure way for you to access your website files on your local computer via a client program such as {{a}}Filezilla{{/a}}. ' +
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **1** times:
`translate( 'It’s a secure way for you to access your website files on your local computer via a client program such as {{a}}Filezilla{{/a}} For more information see {{supportLink}}SFTP on WordPress.com{{/supportLink}} ' )` *ES Score: 10*